### PR TITLE
VM: Implemented SIGN, NEGATE opcode

### PR DESCRIPTION
**Pull request by dauTT**
Closes #231

### CODE REVIEW

File: pkg/vm/vm_ops_maths.go [hunk 1]
```diff
@@ -94,6 +94,47 @@ func Dec(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rst
...
+
+// Negate flips the sign of the stack Item.
+func Negate(op stack.Instruction, ctx *stack.Context, istack *stack.Invocation, rstack *stack.RandomAccess) (Vmstate, error) {
+
+	i, err := ctx.Estack.PopInt()
+	if err != nil {
+		return FAULT, err
+	}
+
+	a := big.NewInt(0)
```
Comment (decentralisedkev) [inline pkg/vm/vm_ops_maths.go]: Here, could you do:
```
	a := big.NewInt(0).Neg(i.Value())
	b, err := stack.NewInt(a)
```

**Comment by decentralisedkev** [comment]
LGTM

File: pkg/vm/vm_ops_maths.go (hunk 1 above)
Comment (dauTT) [reply pkg/vm/vm_ops_maths.go]: :+1:

**Comment by decentralisedkev** [comment]
Waiting for Travis to pass

**Comment by hal0x2328** [comment]
100 points dauTT

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/630c74684cc6dad330397c63e8c7aa6403c043cc1f241638cff87e3b97a40716

**Pull request merged**